packages/rendermime-interfaces/src/index.ts

```diff
@@ -450,4 +444,157 @@ export namespace IRenderMime {
      */
     render(source: string): Promise<string>;
   }
+
+  // ********************** //
+  // Translation interfaces //
+  // ********************** //
+
+  /**
+   * Bundle of gettext-based translation functions for a specific domain.
+   */
+  export type TranslationBundle = {
```

Comment:
Was this snippet copied manually?

Wondering whether there is a risk of it going out of sync later. Although it's probably unlikely this type will be changing any time soon, but it could.

Maybe we should add some logic to the integrity script to make sure they don't drift apart. Haven't looked into how feasible this is, but we could also track that separately if it's too much effort.

Reply:
As `@jupyterlab/translation::TranslationBundle` is proxying this one (as well as `ITranslator`). I don't think there is a risk of divergence.

Reply:
Ah indeed looks good :+1:  The declaration has been updated to:

```ts
export interface ITranslator extends IRenderMime.ITranslator {}
```